packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.scss

```diff
@@ -19,7 +19,7 @@ $ContextualMenu-header-color: $menuHeaderColor;
 
 $ContextualMenu-divider-color: $menuDividerColor;
 
-$ContextualMenu-text-disabled: $disabledTextColor;
+$ContextualMenu-text-disabled: $menuTextDisabledColor;
```

Comment:
yes, it aligns with what I have in other files
(and we'll have to design review it soon anyways)